src/pages/ruleDocumentation.tsx

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { Grid, Typography } from "@mui/material";
+import ValidationRuleCard from "../components/ValidationRuleCard";
+import { localRules, remoteRules } from "../lib/validationRules";
+
+//
+export default function ClientIdentifierDocumentation() {
+  const rules = [...localRules, ...remoteRules];
+  const rulesSorted = rules.sort((rule1, rule2) =>
+    rule1.rule.name.localeCompare(rule2.rule.name)
+  );
```

Comment:
Would it make sense to keep these separates at a high level, so have the docs page explicitly separating the remote and local rules to help distinguishing between issues with the content vs issues with the hosting?